frontend/src/pages/MetricsApplication/utils.ts

```diff
@@ -15,3 +18,41 @@ export const getMetricsApplicationKey = (
 
 	return MetricsApplicationTab.OVER_METRICS;
 };
+
+export const onSaveApDexSettings = ({
+	thresholdValue,
+	refetch,
+	mutateAsync,
+	notifications,
+	handlePopOverClose,
+	servicename,
+}: OnSaveApDexSettingsProps) => (): void => {
+	if (thresholdValue > 0 && thresholdValue < 1 && refetch) {
+		mutateAsync({
+			servicename,
+			threshold: thresholdValue,
+			excludeStatusCode: '',
+		})
+			.then(() => {
+				refetch();
```

Comment:
what we are referencing here can we have a better name here

Reply:
Okay.